guides/source/configuring.md

```diff
@@ -593,6 +593,14 @@ config.session_store :my_custom_store
 
 The default store is a cookie store with the application name as the session key.
 
+#### config.silence_healthcheck_path
```

Comment:
```suggestion
#### `config.silence_healthcheck_path`
```

I think this should fix the Lint check in CI, although that error message could definitely be more friendly 🤔